### PR TITLE
add a workspace command to build storybook

### PR DESCRIPTION
**Pull request by nikek**
Adding a command for build systems to build storybook. Makes the command for services shorter and less fragile for future changes to how we build and from where etc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "create-plugin": "backstage-cli create-plugin",
     "release": "if [ \"$(git symbolic-ref --short HEAD)\" = master ]; then echo \"don't try to release master\"; exit 1; else lerna version --no-push; fi",
     "lerna": "lerna",
-    "storybook": "yarn workspace storybook start"
+    "storybook": "yarn workspace storybook start",
+    "build-storybook": "yarn workspace storybook install && yarn workspace storybook build-storybook"
```
Comment (Rugvip) [inline package.json:21]: Afaik this is the same as just running `yarn install && yarn workspace storybook build-storybook`, how come we want that here? ^_>

Comment (Rugvip) [reply package.json:21]: Can't imagine service providers don't support yarn install + yarn build, it's a really common pattern O_o

Comment (nikek) [reply package.json:21]: Was the best I could come up with to trigger a PR to check if it works. Also thought that having `yarn run build-storybook` lets us switch out the command without changing how netlify in this case runs the command. But I'll close it, seems to work.

Comment (Rugvip) [reply package.json:21]: Yep I'm not against having build-storybook in the root pkg as long as it provides some value. Thinking that currently users should be using the storybook command instead, and CI can easily execute the command in the pkg.  Was more the `yarn install` I was questioning :grin:

**Pull request closed without merging**